### PR TITLE
Fix fit entry parsing.

### PR DESCRIPTION
**Pull request by ChriMarMe**
Fix for #223 

Signed-off-by: Christopher Meis <christopher.meis@9elements.com>

### CODE REVIEW

**Review by xaionaro — APPROVED** [review]
This is great :)

**Review by xaionaro — CHANGES_REQUESTED** [review]
Whoops, forgot to check one thing. Few minutes.

**Review by xaionaro — DISMISSED** [review]

**Review by xaionaro — DISMISSED** [review]

**Review by xaionaro — DISMISSED** [review]

File: pkg/intel/metadata/fit/entry_headers.go [hunk 1]
```diff
@@ -204,6 +209,10 @@ func (hdr *EntryHeaders) getDataCoordinates(firmware io.ReadSeeker, firmwareLeng
 		if err != nil {
 			return nil, nil, nil, []error{err}
 		}
+	case EntryTypeDiagnosticACModuleEntry:
+		panic("not implemented, yet")
```
Comment (walterchris) [inline pkg/intel/metadata/fit/entry_headers.go:213]: is panic the right way to go here?

Comment (xaionaro) [reply pkg/intel/metadata/fit/entry_headers.go]: Fixed.

**Review by xaionaro — APPROVED** [review]

**Review by walterchris — DISMISSED** [review]

**Review by xaionaro — APPROVED** [review]

**Comment by xaionaro** [comment]
@walterchris : Fixed a commend and it dismissed your review decision. Could you repeat please? :)

**Review by walterchris — APPROVED** [review]
Sorry rebased too early - sorry for the noise.

**Pull request merged**